client/src/featureform/serving.py

```diff
@@ -67,6 +69,13 @@ class ServingClient:
     """
 
     def __init__(self, host=None, local=False, insecure=False, cert_path=None):
```

Comment:
probably should add the super init here

Reply:
Maybe I'm missing something important, but if `ServingClient` doesn't inherit from another class, what purpose would calling `super` have here?

Reply:
I think you're fine to leave it out, mixed answers from my reading